server/resources/resource_management.go

```diff
@@ -368,6 +368,70 @@ func CreateOrUpdateResource(
 	return false, UpdateResource(context, dataStore, identityService, resourceSchema, resourceID, dataMap)
 }
 
+func hasProperty(s *schema.Schema, propertyID string) bool {
```

Comment:
This method should be under schema package.

schema.hasProperty(propertyID string)